### PR TITLE
fix and minor modify `format_ae_specific`

### PR DESCRIPTION
**Pull request by LittleBeannie**

### CODE REVIEW

File: R/format_ae_specific.R [hunk 1]
```diff
@@ -43,12 +43,16 @@ format_ae_specific <- function(outdata,
...
   display <- tolower(display)
-
   display <- match.arg(display,
-    c("n", "prop", "total", "diff", "diff_ci", "p", "dur", "mean", "events"),
+    c("n", "prop", "total", "diff", "diff_ci", "diff_p", "dur", "events"),
     several.ok = TRUE
   )
 
+  # Add "n"
+  if(!"n" %in% display){
```
Comment (fb-elong) [inline R/format_ae_specific.R:52]: How about 

`display <- unique(c("n", display))` ?

**Review by fb-elong — APPROVED** [review]

File: R/format_ae_specific.R (hunk 1 above)
Comment (LittleBeannie) [reply R/format_ae_specific.R:52]: Agree! See updated PR below.

Comment (fb-elong) [reply R/format_ae_specific.R:52]: After this change, you do not need the if statement.

**Review by fb-elong — APPROVED** [review]

**Review by BrianLang — APPROVED** [review]
Looks good, thanks! Having this work properly helps with checking the other functions which pipe to/from it.

**Pull request merged**